instance/migrations/0141_features_to_features_extras.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.19 on 2021-05-28 08:56
+
+from django.db import migrations
+import yaml
+
+def forward(apps, schema_editor):
+    """
+    Replace EDXAPP_FEATURES with EDXAPP_FEATURES_EXTRA.
+    """
+    OpenEdXAppConfiguration = apps.get_model('instance', 'OpenEdXAppConfiguration')
+    for config in OpenEdXAppConfiguration.objects.all():
+        settings = (
+            yaml.yaml.safe_load(config.configuration_extra_settings)
+            if config.configuration_extra_settings
+            else {}
+        )
+        if "EDXAPP_FEATURES" in settings and "EDXAPP_FEATURES_EXTRA" not in settings:
+            settings["EDXAPP_FEATURES_EXTRA"] = settings.pop("EDXAPP_FEATURES")
+            settings["EDXAPP_FEATURES_MIGRATED"] = (
+                "EDXAPP_FEATURES was renamed to EDXAPP_FEATURES_EXTRA to preserve previous behavior."
+            )
+            config.configuration_extra_settings = yaml.safe_dump(settings, default_flow_style=False)
+            config.save()
+        
+
+def bacward(apps, schema_editor):
+    """
+    Replace EDXAPP_FEATURES_EXTRA with EDXAPP_FEATURES.
+    """
+    OpenEdXAppConfiguration = apps.get_model('instance', 'OpenEdXAppConfiguration')
+    for config in OpenEdXAppConfiguration.objects.all():
+        settings = (
+            yaml.yaml.safe_load(config.configuration_extra_settings)
+            if config.configuration_extra_settings
+            else {}
+        )
+        if (
+            "EDXAPP_FEATURES_EXTRA" in settings
+            and "EDXAPP_FEATURES" not in settings
+            and settings.get("EDXAPP_FEATURES_MIGRATED")
+        ):
+            settings["EDXAPP_FEATURES"] = settings.pop("EDXAPP_FEATURES_EXTRA")
+            config.configuration_extra_settings = yaml.safe_dump(settings, default_flow_style=False)
+            config.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0140_update_openstack_base_image_20210315_1545'),
+    ]
+
+    operations = [
```

Comment:
FYI @kewne @viadanna There's no operations here, so the forward/backward code doesn't run?

Reply:
Good catch! I readded this after deleting to try another approach and forgot to add the `RunPython`.
It's fixed now.